testsuite/tests/fma/fma.ml

```diff
@@ -0,0 +1,472 @@
+(* TEST *)
+
+(* modified glibc's fma() tests *)
+
+let error l x y z r c =
+  Printf.fprintf stdout
+                 "%s FAIL!\tfma (%h, %h, %h) returned %h instead of %h.\n"
+                 l x y z c r
+
+let success l =
+  Printf.fprintf stdout "%s OK!\n" l
+
+let fma_test l x y z r =
+  let c = Float.fma x y z in
+  if c <> r
+  then error l x y z r c
+  else success l
+
+(* tests 254, 257--259, 264--265, 267--268, and 272--273 are disabled
+   because of unresolved double-rounding issue in FMA emulation
+   fallback *)
```

Comment:
I'll let you decide, but perhaps, for those tests, you could specify either a list of possible results (the correct one, and the one obtained by the emulation), or do a check "modulo epsilon"; just to make sure that the function is not too crazy in those cases.

Reply:
Fixed (with list of possible results), integration in progress, I'm waiting for mscv64 failures if any.